src/renderer/main.tsx

```diff
@@ -7,10 +7,9 @@ import thunkMiddleware from 'redux-thunk';
 import * as actions from './actions'
 import ApplicationPage from './containers/ApplicationPage'
 import { requireElectron } from './electron';
-import { stateReducer } from './reducers';
 import { State } from './state';
-// import {loadPreferences, updatePreferences} from "./actions";
-
+import { stateReducer } from './reducers';
+import {updatePreferences} from "./actions";
```

Comment:
```suggestion
import { updatePreferences } from "./actions";
```

Please update your WebStorm formatting rules!